db-migrations/src/main/resources/db/changelog/sql/storage.sql

```diff
@@ -0,0 +1,17 @@
+--liquibase formatted sql
+
+--changeset ivan.vakhrushev:2024.08.04:storage.table
```

Comment:
Need to add tests with pg-index-health

Reply:
Will be completed within https://github.com/mfvanek/spring-boot-open-telemetry-demo/issues/150